instance/tests/integration/integration_instance.py

```diff
@@ -77,6 +77,31 @@ def f_retry(*args, **kwargs):
     return f_retry  # true decorator
 
 
+def spawn_openstack_server(name_prefix, retries=1):
+    """
+    Spawn an OpenStack server.
+
+    :param name_prefix: Name prefix for OpenStack server
+    :param retries: Number of times it retry spawn a server, defaults tp 1
```

Comment:
typo: "defaults *to* 1"

Reply:
The font of my IDE is not the most appropriate...